.nvmrc

```diff
@@ -0,0 +1,2 @@
+node.js v10.4.1
+npm v6.4.1
```

Comment:
this isn't the correct format for nvmrc file

Reply:
Updated .nvmrc file